src/index.js

```diff
@@ -0,0 +1,130 @@
+// Constants
+const BASE_URL = 'http://127.0.0.1:5000';
+const DEFAULT_CITY = 'Los Angeles';
+const increaseTemp = document.getElementById('increaseTempControl');
+const decreaseTemp = document.getElementById('decreaseTempControl');
+const tempValue = document.getElementById('tempValue');
+const landscape = document.getElementById('landscape');
+const cityNameInput = document.getElementById('cityNameInput');
+const cityNameReset = document.getElementById('cityNameReset'); 
+const headerCityName = document.getElementById('headerCityName');
+const currentTempButton = document.getElementById('currentTempButton');
+
+// Changes color + landscape by Temperature
+let temp = 60;
+const changeTemp = () => {
+    let landscapeEmojis = '';
+    if (temp >= 80) {
+        tempValue.style.color = 'red';
+        landscapeEmojis = '🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂';
+    } else if (temp< 80 && temp >= 70) {
+        tempValue.style.color = 'orange';
+        landscapeEmojis = '🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷';
+    } else if (temp < 70 && temp >= 60) {
+        tempValue.style.color = 'yellow';
+        landscapeEmojis = '🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃';
+    } else if (temp < 60 && temp >= 50) {
+        tempValue.style.color = 'green';
+        landscapeEmojis = '🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲';
+    } else {
+        tempValue.style.color = 'teal';
+        landscapeEmojis = '🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲';
+    }
+    landscape.textContent = landscapeEmojis;
+};
+
+// Changes sky by selection
+const changeSky = () => {
+    const selectedSky = document.getElementById('skySelect').value;
+    const theSky = document.getElementById('sky');
+    let sky = '🌞😎🌞';
+    if (selectedSky === 'sunny') {
+        sky = '🌞😎🌞';
+    } else if (selectedSky === 'cloudy') {
+        sky = '🌥️🌥️🌥️';
+    } else if (selectedSky === 'rainy') {
+        sky = '🌧️🌧️🌧️';
+    } else if (selectedSky === 'snowy') {
+        sky = '🌨️🌨️🌨️';
+    }
+    theSky.textContent = sky;
+};
+
+// Event Handler Functions
+const kelvintoFahrenheit = (temp) => Math.floor((temp - 273.15) * 1.8 + 32);
+
+const incTempButton = () => {
+   temp++;
+   tempValue.textContent = `${temp}`; 
+};
+
+const decTempButton = () => {
+    temp--;
+    tempValue.textContent = `${temp}`;
+};
+
+const updateHeaderCityName = () => {
+    headerCityName.textContent = cityNameInput.value;
```

Comment:
👀 Rather than updating the header directly from the city input, consider having a variable that tracks the city that we update from the event handler, then call a new function that "publishes" any app values to the UI as appropriate. We could also do this for the sky value. Structuring the app this way gives us a consistent data flow (on event, update state, then re-render the entire UI). It can also make it easier to separate the business logic of our application from the UI (which becomes important in larger applications).